engine/src/main/java/org/pentaho/di/base/Params.java

```diff
@@ -96,12 +96,18 @@ public static class Builder {
     private String base64Zip;
     private NamedParams namedParams;
     private NamedParams customNamedParams;
+    private String bowlPath;
+
+    public Builder bowlPath( String bowlPath ) {
+      this.bowlPath = bowlPath;
+      return this;
+    }
```

Comment:
This seems like it's unused? I don't see anywhere in the PR that uses this value.